spec/data-types-and-values.html

```diff
@@ -8,11 +8,50 @@ <h1>ECMAScript Language Types</h1>
     <emu-clause id="sec-ecmascript-language-types-record-type">
       <h1>The Record Type</h1>
       <p>TBD</p>
+
+      <emu-clause id="sec-recordtostring" aoid="RecordToString">
+        <h1>RecordToString ( _argument_ )</h1>
+        <p>The abstract operation RecordToString takes argument _argument_ (a Record) and convertes _argument_ to a String. It performs the following steps when called:</p>
+        <emu-alg>
+            1. Assert: Type(_argument_) is Record.
+            1. Return *"[record]"*.

```

Comment:
that's it? no string version of the record, for debugging?

Reply:
Objects return `[object Object]`, if you want it to be `[object Record]`, that's at least defensible.

For arrays, toString returns the contents, which is better than nothing - but either way this seems like an opportunity to do *better* than objects and arrays.

Reply:
Wanna open up an issue to track discussion? I'd be fine with `[object Record]` here as a starting point, personally.

Reply:
Filed #136.